src/pages/Main/BannerItem/BannerItem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import BANNERS_DATA from '../BANNERS_DATA';
+
+function BannerItem() {
+  return (
+    <Item>
+      {BANNERS_DATA.map(BANNER => (
```

Comment:
```suggestion
      {BANNERS.map(banner => (
```

인자로 들어오는 값은 계속 바뀌기 때문에 상수로 사용하지 않습니다. 그리고 _DATA 등의 suffix는 굳이 안붙여도 되니까 안붙이는것도 좋습니다!

그리고 매개변수자리에서 banner 바로 구조분해할당하는것도 괜찮아보이네요!

Reply:
배너 부분은 바뀌는 데이터가 아니라서 상수데이터로 두었는데 수정해야할까요?

Reply:
callback의 인자는 변하는 값이니까 상수로 네이밍하는건 좋지 않을 것 같네요!